docs/source/Advanced-Configuration.md

```diff
@@ -67,6 +68,12 @@ While configuring Google Auth2 you can choose either to enable access to the all
 <a href="https://support.google.com/cloud/answer/10311615#user-type&zippy=%2Cinternal%2Cexternal" target="_blank">Reference</a>
 </div>
 
+#### GitHub OAuth
+
+Create a new OAuth app on [GitHub](https://github.com/settings/applications/new), set the `Authorization callback URL` as the `http://localhost/api/auth/github-callback` and then generate the `Client ID` and `Client secret`.
```

Comment:
That is not entirely correct, because localhost would work only if you are running the application locally. We should say that `http://<your_domain|localhost>/api/auth/github-callback` instead